pages/account/signup.vue

```diff
@@ -79,7 +77,7 @@ const submit = async () => {
         }
       }
     } else {
-      setUser(data.value.user)
+      setUser(data.value.user ? data.value.user : {"username": "unverified-user"} )
```

Comment:
Is there other code that tests against this magic `unverified-user` string? What happens if I try to create an account named `unverified-user`?

Reply:
The only code that tests against this should be the auth middleware in the front end. I added this so that the middleware doesn't redirect you to the sign-in page but to the onboarding page.

Another option would have been modifying the backend registration endpoint so that it would return the unverified user data, which doesn't happen by default.